src/language/walker/programAwareRuleWalker.ts

```diff
@@ -20,8 +20,8 @@ import {IOptions} from "../../lint";
 import {RuleWalker} from "./ruleWalker";
 
 export class ProgramAwareRuleWalker extends RuleWalker {
-    protected program: ts.Program;
-    protected typeChecker: ts.TypeChecker;
+    private program: ts.Program;
```

Comment:
@JKillian is there a reason your style preference is for field declaration rather than a `private